CHANGELOG.md

```diff
@@ -4,6 +4,14 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.80.2-alpha.1
+
+No longer will prioritize the component names for replays, as this will break searching by CSS selector.
```

Comment:
What is the long term plan for replays then? Don't we need this change for them also?